src/documents/proofOfAddress/proofOfAddressV1.ts

```diff
@@ -0,0 +1,113 @@
+import { Document, DocumentConstructorProps } from "../document";
+import {
+  CompanyRegistration,
+  DateField,
+  Locale,
+  TextField,
+} from "../../fields";
+
+export class ProofOfAddressV1 extends Document {
+  /** ISO date yyyy-mm-dd. Works both for European and US dates. */
```

Comment:
Technically it's US vs the entire rest of the world (and possibly beyond), but OK.

Reply:
FYI, I copy paste the docs from the Open API documentation